account_consolidation/wizard/consolidation_check_view.xml

```diff
@@ -10,17 +10,17 @@
                 <field name="state" invisible="True" />
                 <div states="open">
                     <label string="Prepare your consolidation and make sure that your consolidation will be correct by proceeding with the checks." />
-                    <group>
+                    <group name="company">
                         <field name="company_id" />
                     </group>
-                    <group>
+                    <group name="checks">
                         <p>Check accounts : Check consistency between subsidiaries accounts and Holding company consolidation accounts." </p>
                         <p>Check partners : Check that partners linked to subsidiaries have no company defined." </p>
                         <p>Check companies allowed : Check that user has access to subsidiaries." </p>
                     </group>
                 </div>
                 <div states="error,ok">
-                    <field name="message" nolabel="1" />
+                    <field name="message" nolabel="1" class="alert alert-[warning][-danger][-info]" />
```

Comment:
@grindtildeath ehm, you have to pick one class only: `alert-warning` (yellow) or `alert-danger` (red) or `alert-info` (blueish) or `alert-success` (green) based on the kind of message you are displaying. Like:
```xml
<field name="message" nolabel="1" class="alert alert-danger" states="error"/>
<field name="message" nolabel="1" class="alert alert-success" states="ok"/>
```